az-digital/az_quickstart
# Closes #2671: Enable copy options based on content type

## Description
Make "Copy email snippet" options pertinent to content type. Removed fields that were left in case they were needed for this purpose.

## Related issues
Closes #2671

## How to test
1. Enable Quickstart for Marketing Cloud
2. Verify event and news copy links show as expected
3. Ensure copy functionality still works

### Example
![image](https://github.com/az-digital/az_quickstart/assets/59745109/51269988-298b-4fa8-bb71-33d29b337968)


## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
